### ISSUE TITLE
Needs a simple 'Getting Started' kind of doc

### ISSUE THREAD

**Issue by furicle** [twcloud/tw5-dropbox#5]
Something short and sweet like,

1. Download an empty Tiddly Wiki 
2. Visit https://twcloud.github.io/tw5-dropbox/ and authorize 
3. Put the TW file in the apps folder if you chose apps, or anywhere if you didn't
4. Reload the https://twcloud.github.io/tw5-dropbox/ page and click on your tiddly wiki file
5. Repeat step 4 whenever you like
6.  What do you do if you don't have online access?
7. How can you self host/create your own app rather than use twoclould.github.io?

**Comment by mfncooper** [comment]
Yes, indeed. Until I found this issue, with the list of steps, I had no idea how to proceed. The TiddlyWiki Getting Started just says to "choose a method for saving changes", but once I choose TiddlyWiki Cloud Connectors, there is no indication of what I'm supposed to do next, either on the TiddlyWiki page or the page linked from it. There is also no contact information so that I could ask, hence my adding to this issue. The page at https://twcloud.github.io/ describes 3 different versions, but doesn't really explain why I would choose one over another (or why the author has created 3 different ones). None of them appear to have been touched for 3 or 4 years, so I'm not sure if it's safe to choose any of them - the whole project looks like it might be dead. If I click on the first one anyway, I'm then presented with a choice of "Full Dropbox Access" or "Apps Folder Access", but with no explanation of what these are, what they mean to me, or why I would choose one over the other. This is really too bad. I really like the idea of using TiddlyWiki, and having it in my Dropbox so I can use it from multiple devices. But without documentation, and an indication that it's a supported project, perhaps another "method for saving changes" might be a safer choice.
